src/components/Newsletter/Newsletter.js

```diff
@@ -145,41 +117,12 @@ class Newsletter extends React.Component {
             name="cm-f-jrdtwd"
             value={countyName || ''}
           />
-          <Autocomplete
-            multiple
-            id="alert-locations"
-            defaultValue={this.defaultValues}
-            getOptionSelected={(option, value) =>
-              option.full_fips_code === value.full_fips_code
-            }
-            onChange={(event, newValue) => {
-              this.handleSelectChange(newValue);
-            }}
-            options={this.autocompleteOptions}
-            getOptionLabel={option =>
-              option.county
-                ? `${option.county}, ${option.state_code}`
-                : option.state
-            }
-            renderTags={(tagValue, getTagProps) =>
-              tagValue.map((option, index) => (
-                <Chip
-                  label={
-                    option.county
-                      ? `${option.county}, ${option.state_code}`
-                      : option.state
-                  }
-                  {...getTagProps({ index })}
-                />
-              ))
-            }
-            renderInput={params => (
-              <TextField
-                variant="outlined"
-                {...params}
-                placeholder="+ Add alert locations"
-              />
-            )}
+          <AutocompleteRegions
+            regions={regions.all()}
+            selectedRegions={selectedRegions}
+            onChangeRegions={onChangeRegions}
+            ariaLabelledBy=""
```

Comment:
should this be empty?

Reply:
not really 😅  - I added a default label so it should be good even if we don't have a label for the element